packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -23,6 +23,13 @@ import {
 import * as Classes from "../../common/classes";
 
 export interface ITagInputProps extends IProps {
+    /**
+     * If true, onAdd will also be invoked when the input loses focus.
```

Comment:
Don't think the word `also` is necessary here, especially since this is the first prop in the list. I think it reads more clearly without it.

Reply:
Fixed